### ISSUE TITLE
"Could not save bundle.can't be blank" during mist bundle installation in latest docker-compose version

### ISSUE THREAD

**Issue by MichaelPereira** [operable/cog#938]
Hi,

When deploying cog using the docker compose file with Docker for Mac (not docker-machines) from the master branch, I am unable to install the mist bundle:
```
$ cogctl bundle install net/config.yaml  --enable --relay-groups="default"
cogctl: ERROR: "Could not save bundle."
cogctl: ERROR: "can't be blank"
```

It doesn't show up in the list of bundles:
```
$ cogctl bundle -v
NAME      ENABLED VERSION  INSTALLED VERSIONS  BUNDLE ID
format    0.4.2            0.4.2               6d88ee4f-62b8-4f53-ac8a-9ad85896d128
net       0.1.0            0.1.0               64a6d561-cbee-46e8-bb83-3fb143d41d68
operable  0.12.0           0.12.0              6e90dc86-6dac-487e-9218-8db24bee7ede
```

The error is different if I retry to install it:
```
$ cogctl bundle install net/config.yaml  --enable --relay-groups="default"
cogctl: ERROR: "Could not save bundle."
cogctl: ERROR: "version has already been taken"
```

**Comment by MichaelPereira** [comment]
This problem reproduces only when using the `latest` of the docker images, it doesn't appear with `0.12`

**Comment by vanstee** [comment]
I'm having trouble reproducing this. While running Cog ([d784ba7](https://github.com/operable/cog/blob/d784ba77a2419f72a2a3ffe9a9a5bdbffe4894d1/docker-compose.yml) on Docker for Mac I was able to install the Mist bundle ([572893c](https://github.com/cogcmd/mist/blob/572893ce9c02d7d6c58a0611dddd258b7e9c17b6/config.yaml)).

This was probably just a bad copy-paste, but your examples show you installing the Net bundle. Just wanted to confirm you were seeing this error with the Mist bundle, right?

Either way I need to improve the error message we return from cogctl. Thanks for creating an issue.

**Comment by vanstee** [comment]
I'm going to mark this as closed. But feel free to open if you can repro with more details.

**Comment by utsysadmin** [comment]
I'm having the exact same issue installing my config.yml using Docker for mac and the compose files.

config.yml
`
---
cog_bundle_version: 4
name: marathon
description: "Deploy Utility"
version: "0.0.1"
docker:
  image: docker-image
  tag: latest
permissions:
  - marathon:deploy
  - marathon:prod
commands:
  deploy:
    description: "Deploy"
    executable: "/deploy.py"
    arguments: "--deploy"
    options:
      env:
        description: "select environmnet"
        type: string
        required: true
        short_flag: e
      apps:
        description: "select app"
        type: string
        short_flag: a
      group:
        description: "select group"
        type: string
        short_flag: g
    rules:
      - when command is marathon:deploy must have marathon:deploy
`
The result:
`
bash-4.3$ cogctl bundle install config.yml --enable --relay-group=default
Error: Could not save bundle. can't be blank
`
I've tried pruning my docker system and starting over.  After the system came up I ran docker-compose exec cog bash from another terminal and installed a user and chat-handle.  After I tried to install my bundle.
